### PR TITLE
Functional tests - Create edit view voucher in customer information page

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Create edit view voucher in customer information page
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/04_customers/01_customers/09_viewEditVoucher" URL_FO=httpyourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18843)

### CODE REVIEW

File: tests/puppeteer/campaigns/data/faker/cartRule.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const faker = require('faker');
+
+module.exports = class Category {
```
Comment (SimonGrn) [inline tests/puppeteer/campaigns/data/faker/cartRule.js]:
```suggestion
module.exports = class CartRule  {
```
Am I misundestanding this class ?

Comment (nesrineabdmouleh) [reply tests/puppeteer/campaigns/data/faker/cartRule.js]: You're right :) done thank you

File: tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/09_viewEditVoucher.js [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    addAddressPage: new AddAddressPage(page),
+    viewCustomerPage: new ViewCustomerPage(page),
+    viewOrderPage: new ViewOrderPage(page),
+    cartRulesPage: new CartRulesPage(page),
+    addCartRulePage: new AddCartRulePage(page),
+  };
+};
+
+// Create edit voucher from view customer page
+describe('Create/ Edit and view voucher from customer information page', async () => {
```
Comment (Progi1984) [inline tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/09_viewEditVoucher.js]:
```suggestion
describe('Create/Edit and view voucher from customer information page', async () => {
```

Comment (nesrineabdmouleh) [reply tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/09_viewEditVoucher.js]: Done thank you :)

**Review by Progi1984 — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Pull request merged**